.github/workflows/build_and_deploy_ext2_img.yml

```diff
@@ -0,0 +1,119 @@
+name: Create and deploy ext2 image to pages
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+  # Lets you supply arguments when you run a github action manually.
+    inputs:
+      dockerfile_context_path:
+        description: 'Path or url to the Dockerfile its context.'
+        required: true
+        default: 'dockerfiles'
+
+      dockerfile_path:
```

Comment:
It sounds cumbersome to manually provide both the file and the directory. It should be possible to get the directory from the file instead

Reply:
The Dockerfile context path and the Dockerfile_path are not serving the same purpose. One points to the Dockerfile, the other points to the build context that we send to the Docker daemon. To my understanding it is not possible to achieve this through the Dockerfile itself.

If we omit the `--file` option, the Dockerfile would always need to be named Dockerfile and it would always attempt to grab /PATH/Dockerfile. It would give us much less freedom.

In hindsight I should've named the variable docker_build_context.

Should I remove the option and only keep path?